src/control-flow-basics/exercise.rs

```diff
@@ -25,15 +25,14 @@ fn collatz_length(mut n: i32) -> u32 {
     len
 }
 
-// ANCHOR: tests
-#[test]
-fn test_collatz_length() {
-    assert_eq!(collatz_length(11), 15);
-}
-// ANCHOR_END: tests
-
 // ANCHOR: main
 fn main() {
-    println!("Length: {}", collatz_length(11));
+    println!("Length: {}", collatz_length(11)); // should be 15
 }
 // ANCHOR_END: main
+// ANCHOR_END: solution
+
+#[test]
+fn test_collatz_length() {
+    assert_eq!(collatz_length(11), 15);
+}
```

Comment:
This is excluding the test from the starting code and solution, was that intended? I thought the goal was to normalize tests on using tests instead of `main`.